ci/test-install/Makefile

```diff
@@ -21,7 +21,7 @@
 DEPS       := bigtable_client google_cloud_cpp_common grpc++ grpc zlib openssl libcares protobuf
 CXXFLAGS   := $(shell pkg-config $(DEPS) --cflags)
 CXXLDFLAGS := $(shell pkg-config $(DEPS) --libs-only-L)
-LIBS       := $(shell pkg-config $(DEPS) --libs-only-l) -lgpr $(shell pkg-config libcares --libs-only-l)
+LIBS       := $(shell pkg-config $(DEPS) --libs-only-l) -lgpr -laddress_sorting $(shell pkg-config libcares --libs-only-l)
```

Comment:
Why this change?

Reply:
Because gRPC installs incomplete pkg-config files:

https://github.com/grpc/grpc/issues/15166

Let me add an explanation for future readers.

Reply:
Perfect, thanks!